### DISCUSSION TITLE
App constantly leads to same error window

### DISCUSSION THREAD

**Discussion by guidoooo** [TumblThreeApp/TumblThree#387] [General]
Hello! I'm using Windows 10 on a 64 bit computer, my browser is the latest Firefox version available, and whenever I've tried using TumblThree (either when first booting up, or at most when entering a URL on the bottom left space) I consistently, and repeatedly, to no avail, get the following screen. Rebooting not reinstalling have caused any improvement. When this screen shows up, nothing can be clicked (not even the button on the bottom left to alert the developper) and I have to forcefully shut down the program from the task manager.

![Screenshot 10 11 22 - 15 34](https://user-images.githubusercontent.com/90530683/201765462-05a0ad3b-e5ad-468e-9adf-a9653b20aec1.PNG)

I have read other discussions but seems like nobody has mentioned anything like this so I'm kinda lost on what to do haha. Thank you for reading!

**Comment by thomas694** [comment]
You can have a look in the log file (%LOCALAPPDATA%\TumblThree\TumblThree.log), whether you can find an error at the end of the file.

**Comment by guidoooo** [comment]
Thanks for the quick response Thomas! The specified log file is a 48MB text with over 485k lines of text haha. I'm not smart enough to understand any of this. If it's too bothersome to continue with this issue I won't waste your time. Thank you!

**Comment by thomas694** [comment]
No problem. Delete the old log file and start the app again. As you said the error comes immediately. Close it and look for a block of lines at the end of the file starting with a date and time. Paste it here or send it by email.

**Comment by guidoooo** [comment]
That makes it a much more pallatable size haha. Uploading it now!
[TumblThree.log](https://github.com/TumblThreeApp/TumblThree/files/10017051/TumblThree.log)

**Comment by thomas694** [comment]
You have 64 bit windows and the app is the 64 bit version, but even the 32 bit version would run then. The app can start and write the log file, but when opening the GUI it crashes with a `BadImageFormatException` (PresentationFramework-SystemData). That could mean that something with the .NET Framework installation got corrupted. Can you run other .NET applications?
Can you try to run the `TumblThreeImageViewer.exe` in the TumblThree folder?
Does the 32 bit version of TumblThree work (you can download and extract this zip to a separate folder for testing)?
